src/main/java/ru/itis/configuration/Config.java

```diff
@@ -0,0 +1,14 @@
+package ru.itis.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration
+public class Config {
```

Comment:
Your bean is duplicated, delete this file.

Reply:
fixed